.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is.
+
+## To Reproduce
+Steps to reproduce the behavior:
+1. Run `...`
+2. Use this custom-resource.yaml from this repository or gist
+3. Run this operator command `...`
+4. See error
+
+## Expected behavior
+A clear and concise description of what you expected to happen.
+
+## Operator execution and Output
+```
+$ kubectl apply -f <porter-resource>.yaml
+lots of output
+```
+
+## Version
+Please provide the porter-agent and the operator version number that's used to reproduce the bug
```

Comment:
Let's ask for their AgentConfig since that will give us the versions and all the extra info we need to debug.